lib/mixlib/install/product.rb

```diff
@@ -193,8 +193,13 @@ def lookup(key, version = :latest)
 
   product "automate" do
     product_name "Chef Automate"
-    package_name "delivery"
-    ctl_command "delivery-ctl"
+    # Delivery backward compatibility
+    package_name do |v|
+      v < version_for("0.7.0") ? "delivery" : "automate"
+    end
+    ctl_command do |v|
+      v < version_for("0.7.0") ? "delivery-ctl" : "automate-ctl"
```

Comment:
Thanks for pointing this out. According to `https://docs.chef.io/ctl_delivery_server.html` we should be ok using version 0.6.0 instead of 0.6.6. I will make this change.
```
Prior to Chef Automate 0.6.0, this tool was named delivery-ctl. To support backwards compatibility, you can continue to call delivery-ctl in Chef Automate 0.6.0.
```